### PR TITLE
Pareto scaler

### PR DESCRIPTION
**Pull request by rbschoop**

### CODE REVIEW

File: data_science/MyParetoScaler.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## MyPareto Scaler 
+import sklearn  # has to be installed via conda install scikit-learn
+
+from sklearn.base import (
+    TransformerMixin,
+    BaseEstimator,
+)
+
+
+class MyParetoScaler(TransformerMixin, BaseEstimator):
```
Comment (JCZuurmond) [inline data_science/MyParetoScaler.py]: Could you remove the `My`?

Comment (rbschoop) [reply data_science/MyParetoScaler.py]: Why? Is there a Standard ParetoScaler command?

File: data_science/MyParetoScaler.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        self.means_ = np.mean(X, axis=1).reshape(-1,1) ## col vector
+        self.stds_ = np.std(X, axis=1).reshape(-1,1)  ## col vector
+        return self
+    
+    def transform(self, X, y=None):
+        if self.with_mean:
+            X = X - self.means_ 
+        if self.with_std:
+            X = X / np.sqrt(self.stds_)  
+
```
[the remark this answers is not in the document]
Comment (rbschoop) [reply data_science/MyParetoScaler.py]: Agree

File: data_science/MyParetoScaler.py (hunk 1 above)
Comment (rbschoop) [reply data_science/MyParetoScaler.py]: ok, done. 
Will push it in a few secs

**Pull request merged**